docs/docs/creating-a-sitemap.md

```diff
@@ -2,7 +2,29 @@
 title: Creating a sitemap
 ---
 
-This is a stub. Help our community expand it.
+### What is a Site Map?
```

Comment:
Why Site Map vs. sitemap? Which is correct?

I think let's be consistent and always go one word, e.g.

```suggestion
### What is a Sitemap?
```